lib/src/protos/op_store.proto

```diff
@@ -67,7 +67,11 @@ message View {
   repeated Tag tags = 6;
   // Only a subset of the refs. For example, does not include refs/notes/.
   repeated GitRef git_refs = 3;
-  bytes git_head = 7;
+  // This field is just for historical reasons (before we had the RefTarget
+  // type). New Views have (only) the target field.
+  // TODO: Delete support for the old format.
+  bytes git_head_legacy = 7 [deprecated = true];
+  RefTarget git_head = 9;
```

Comment:
Better to document the format change in changelog?

Reply:
Users shouldn't really notice it, except that won't see "HEAD@git" if they use an old jj binary after they've used the new one. Do you mean that we should document the possibility of the "HEAD@git" reference having conflicts?

Reply:
The new->old jj binary case. I think it's not uncommon to try old version if something goes wrong after upgrading.

Reply:
Makes sense. I've added a mention of that in the changelog. The effect is actually kind of hard to notice because the new head is imported whenever you run a command with the old binary and the HEAD@git is not there, so it's basically only `jj log --at-op=X` where you may notice the missing reference.